lib/shopify_api/resources/inventory_level.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module ShopifyAPI
+  class InventoryLevel < Base
+
+    # The default path structure in ActiveResource for delete would result in:
+    # /admin/inventory_levels/#{ inventory_level.id }.json?#{ params }, but since
+    # InventroyLevels are a second class resource made up of a Where and a What
+    # (Location and InventoryItem), it does not have a resource ID. Here we
+    # redefine element_path to remove the id so HTTP DELETE requests go to
+    # /admin/inventory_levels.json?#{ params } instead.
+    #
+    def self.element_path(prefix_options = {}, query_options = nil)
+      prefix_options, query_options = split_options(prefix_options) if query_options.nil?
+      "#{prefix(prefix_options)}#{collection_name}.#{format.extension}#{query_string(query_options)}"
+    end
+
+    def destroy
+      load_attributes_from_response(
+        self.class.delete('/', location_id: location_id,
```

Comment:
This should still fit into a 120 char line.

Reply:
the data being sent can be added to the payload body, not necessarily the query params. The backend doesn't care where the content is being sent. 

So if we are close to the 120 char limit, we should move it to the body.

Reply:
can't _quite_ get it to 120 
[![alt](https://screenshot.click/07-03-18-11-11-i206l-kxo9i.jpg)](https://screenshot.click/07-03-18-11-11-i206l-kxo9i.jpg

next best thing? 
[![alt](https://screenshot.click/07-03-18-11-12-rcpxg-b1slg.jpg)](https://screenshot.click/07-03-18-11-12-rcpxg-b1slg.jpg)

@celsodantas `#destroy` will take params or body, but its difficult to send a body with DELETE in ActiveResource. I figure if they both work, stick with query params here and if someone wants to hit the endpoint with cURL or manually, then that's fine too. We can only ever send one location and one item at a time anyway, so its not like the query string will ever be unreasonably long.